dev/com.ibm.ws.org.apache.cxf.cxf.rt.transports.http.3.2/src/org/apache/cxf/transport/http/AbstractHTTPDestination.java

```diff
@@ -268,7 +267,9 @@ public void invoke(final ServletConfig config,
 
         copyKnownRequestAttributes(req, inMessage);
 
-        inMessage.put(HttpServletResponse.class, resp); // Liberty change - reqd for SSE see LibertySseEventSinkImpl
+        //Liberty code change start
```

Comment:
no need for the comment here or on line 272, since there is already a comment on line 271.

Reply:
I think Andy wanted you to leave the comment you just removed and to remove the new comments you added.  The one you just removed has extra details that it is good not to lose.